sequencing/vcf_pca/Readme.md

```diff
@@ -1,21 +1,18 @@
-<!-- dx-header -->
 # Calculate PCs from VCF (DNAnexus Platform App)
 
-Uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF or PLINK bed/bim/fam files.
+## What does this applet do?
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+This app uses PLINK 1.9 and eigenstrat 6.0 to calculate principal components from VCF (`*.vcf.gz`) or PLINK files(`*.bed`, `*.bim`, `*.fam`). The app can process both type of data at the same time.
 
+## What data are required for this app to run?
 
+This app takes the following inputs:
 
-<!--
-TODO: This app directory was automatically generated by dx-app-wizard;
-please edit this Readme.md file to include essential documentation about
-your app that would be helpful to users. (Also see the
-Readme.developer.md.) Once you're done, you can remove these TODO
-comments.
+For VCF data, this app requires equal number of VCF (variant calling format) files(`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`). VCF is the standard format for variant calling. 
+
+For PLINK, this app requires equal number of binary genotype files (`*.bed`), extend mapping files (`*.bim`), and FAM files (`*.fam`) -- first 6 columns of pedigree. PLINK is a standard format for whole genome association analysis of genotype/phenotype data.
```

Comment:
Change to "For PLINK, this app requires an equal number of binary genotype files (`*.bed`), extend mapping files (`*.bim`), and FAM files (`*.fam`) -- first 6 columns of pedigree. PLINK is a standard format for whole-genome association analysis of genotype/phenotype data."

Do we have to explain the PLINK format? Also, what does "first 6 columns of pedigree" mean?

Reply:
I think PLINK is a bit more complicated than VCF, so I wish to explain a bit.
"first 6 columns of pedigree" is the definition of such file. I think I didn't say it well. It should be "first 6 columns of the pedigree file".